js/tests/unit/tooltip.js

```diff
@@ -680,6 +680,42 @@ $(function () {
     $styles.remove()
   })
 
+  test('should not be adjust position when scrolling', function () {
```

Comment:
You are right, I guess more test coverage should be added.  This test was to catch the issue where tooltips were bound inside the browser's window even when the viewport was set as body (when normally they should have remained at the same position, next to their target)